.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ repos:
     -   id: check-executables-have-shebangs
     -   id: check-merge-conflict
 
--   repo: git@github.com:elastic/apm-pipeline-library
+-   repo: https://github.com/elastic/apm-pipeline-library
     rev: current
     hooks:
     -   id: check-bash-syntax
```

Comment:
L13 and L14 are not needed could you update this file 🙏 ?

Reply:
https://github.com/elastic/ecs-logging-php/pull/41/commits/7dcd5384604b23918489d56b99d299996c081d9f